### ISSUE TITLE
Dutch: Several translation strings are missing

### ISSUE THREAD

**Issue by LukeSerne** [plk/biblatex-apa#224]
**Description**
The dutch language file is missing quite a few translation strings. I figured I'd submit the translations and abbreviations I used, based on the [localisation by the Tilburg University](https://libguides.uvt.nl/apa-handleiding/afkortingen). This also includes a correction: `origineel werk gepubliceerd` -> `originele werk gepubliceerd`.

I also added two new translation strings (`videopodcast` and `videopodcastepisode`), since the aforementioned localisation distinguishes between audio and video podcasts. Finally, I also updated the `mainaudio` bibmacro to handle `videopodcastepisode` like `podcastepisode`, and use "in" instead of "on".

**Code**
```latex
\NewBibliographyString{videopodcast}
\NewBibliographyString{videopodcastepisode}

\DefineBibliographyStrings{dutch}{
    jourarticle={artikel},
    present={heden},
    video={video},
    host={host},
    hosts={hosts},
    podcast={audiopodcast},
    podcastepisode={audiopodcastaflevering},
    origyear={originele\space werk\space gepubliceerd},
    spring={{}voorjaar},
    summer={{}zomer},
    autumn={{}herfst},
    winter={{}winter},
    springN={{}voorjaar (noordelijk halfrond)},
    summerN={{}zomer (noordelijk halfrond)},
    autumnN={{}herfst (noordelijk halfrond)},
    winterN={{}winter (noordelijk halfrond)},
    springS={{}voorjaar (zuidelijk halfrond)},
    summerS={{}zomer (zuidelijk halfrond)},
    autumnS={{}herfst (zuidelijk halfrond)},
    winterS={{}winter (zuidelijk halfrond)},
    Q1={{}eerste kwartaal},
    Q2={{}tweede kwartaal},
    Q3={{}derde kwartaal},
    Q4={{}vierde kwartaal},
    QD1={{}eerste tertaal},
    QD2={{}tweede tertaal},
    QD3={{}derde tertaal},
    S1={{}eerste semester},
    S2={{}tweede semester},
    videopodcast={videopodcast},
    videopodcastepisode={videopodcastaflevering},
}

\renewbibmacro*{mainaudio}{%
  \iffieldundef{maintitle}{}
    {
     \ifthenelse{\iffieldequalstr{entrysubtype}{podcastepisode}\OR\iffieldequalstr{entrysubtype}{videopodcastepisode}}
        {\bibcpstring{in}}
        {\bibcpstring{on}}%
     \setunit{\addspace}%
     \printfield{maintitle}}}
```

I added `{}` in front of the seasons and `Q?`, `QD?` and `S1` translation items, because that prevents the first character from being capitalised (so we get `(2020, voorjaar)` instead of the incorrect `(2020, Voorjaar)`). If there is a better solution to prevent this capitalisation, that should be used instead.

**Comment by plk** [comment]
Please note that some of the strings had to be fixed in `biblatex` and so you'll need the `dutch.lbx` from the dev branch there to be updated.

**Comment by LukeSerne** [comment]
Thanks for doing all this work to incorporate all these Dutch peculiarities.

Things are looking very good with the new version. The only thing I found was that the `present` key doesn't seem to be used anymore.

**MWE**
```latex
\documentclass{report}

\usepackage[dutch]{babel}
\usepackage[style=apa, backend=biber]{biblatex}

\begin{filecontents}[overwrite]{test.bib}
@online{Date,
    author = {Tes, Terry},
    date = {2001/},
    title = {Open date range},
    url = {https://www.example.com},
}
\end{filecontents}
\addbibresource{test.bib}

\begin{document}
\nocite{*}
\printbibliography
\end{document}
```

**Observed output**
Tes, T. (2001–). _Open date range_. https://www.example.com

**Expected output**
Tes, T. (2001–heden). _Open date range_. https://www.example.com

**Comment by LukeSerne** [comment]
Yeah, that fixes it. Thanks. It's no problem that these formats were not tested that well - unfortunately it seems the majority of people using APA are writing their reports and papers in Word instead of latex (at least in the Netherlands), so it makes sense that these packages do not see much use. However, I really like using latex and think it has many benefits over Word, so I'm glad this style exists and there are people like you that are willing to maintain it.

What does the release schedule for `biblatex-apa` and `biblatex` look like? As part of testing these packages for compliance with a Dutch localisation of APA7, I implemented all examples from a manual in latex, which plan on publishing. Since these patches are required to get the correct output, it would be great if there is a release that incorporates the patches that I can point people to. There's no hurry though, I'm just curious.

**Comment by plk** [comment]
@moewew - do you know where we stand on the L3 changes that were going to force a release on us? If that's not a looming issue, I am leaning towards releasing 2.20/3.20 and APA fairly soon.

**Comment by moewew** [comment]
Funnily enough Joseph just asked people about their opinion on the L3 changes again in https://github.com/latex3/latex3/pull/1247. As I wrote there, I haven't yet looked at the changes we'd have to do under the new plan. But I think we should try to get that in before we release, so I will try to see what I can do in the next week.
